### ISSUE TITLE
[PRL22 Visual Bug] Original Clean: Bad squarebeams shadows on exit corridors if "Fast Light" was selected

### ISSUE THREAD

**Issue by LautaroL20** [BEEmod/BEE2-items#896]
![20161005090852_1](https://cloud.githubusercontent.com/assets/12244376/19112633/40203f20-8adc-11e6-89f8-e24726c093f6.jpg)

**Comment by TeamSpen210** [comment]
This particular squarebeams issue actually only happens when compiling with 'fast' lighting, so it won't appear in published maps.

**Comment by LautaroL20** [comment]
Ah, i forgot to look into the compiling settings to select "Full light".

**Comment by TeamSpen210** [comment]
It still needs to be fixed for fast compiles.

**Comment by LautaroL20** [comment]
In that case, ill fix the post´s title.

**Comment by Luke18033** [comment]
Doesn't really need to be fixed.

**Comment by LautaroL20** [comment]
> Doesn't really need to be fixed.

Yes it does, i saw this same corridor on maps that don´t use Beemod and squarebeams does not look that dark...

**Comment by LautaroL20** [comment]
And i think that this happens regardless if "fast light" is enabled or not.

**Comment by Luke18033** [comment]
Okay, if it happens on full lighting then it is a real issue that needs to be fixed.
